### ISSUE TITLE
Maps and graphs should be reporting rate, not raw numbers

### ISSUE THREAD

**Issue by dbl3raf** [bird-team/brisbane-birds-prototype#10]
I'm sure this is in the development pipelines, but reporting rate is:

# of lists [for that year, grid cell, month etc] with the species of interest / total number of checklists [for that year, grid cell, month etc] where PROTOCOL_TYPE does not equal "eBird - Casual Observation", and ALL_SPECIES_REPORTED = 1

**Comment by jeffreyhanson** [comment]
I wasn't sure what reporting rate was - thanks for saying that explicitly. I'll update it to use that.

Should I use this definition of reporting rate for the "reporting rate vs. month" graph and "reporting rate vs. year" graph, or should the "reporting rate vs. year" graph be "total number of records vs. year"?

**Comment by dbl3raf** [comment]
Yes, this definition applies throughout, i.e. to the maps (grid cells should be shaded according to reporting rate), the monthly and the yearly plots. The only case this doesn't apply is with the elevation, where it's a box plot of elevation across all records, regardless of whether they were submitted in complete or incomplete checklists.

I realised my definition above omits "number" at the beginning - I had typed a gate, but that seems be a reserved symbol "#"

**Comment by jeffreyhanson** [comment]
Also, I forgot to mention, GitHub uses markdown formatting. Here's a cheatsheet that might help](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet).
